AK/Format.h

```diff
@@ -551,10 +551,13 @@ void outln(FILE* file, CheckedFormatString<Parameters...>&& fmtstr, const Parame
 inline void outln(FILE* file) { fputc('\n', file); }
 
 template<typename... Parameters>
-void out(CheckedFormatString<Parameters...>&& fmtstr, const Parameters&... parameters) { out(stdout, move(fmtstr), parameters...); }
+void out(CheckedFormatString<Parameters...>&& fmtstr, const Parameters&... parameters)
```

Comment:
I...don't see how this is a forwarding reference? This should only ever bind to a temporary CheckedFormatString afaik.

Reply:
Ah! You are right. I thought that since `Parameters...` was deduced that `fmtstr` would be deduced. Apparently it is not. Interesting...

https://serenityos.godbolt.org/z/oe8aeqno1

Glad to learn something new today.